tracer/test/Datadog.Trace.Security.IntegrationTests/AspNetWebForms.cs

```diff
@@ -101,7 +101,7 @@ public async Task TestBlockedRequest(string test)
             var url = "/Health";
 
             var settings = VerifyHelper.GetSpanVerifierSettings(test);
-            await TestAppSecRequestWithVerifyAsync(_iisFixture.Agent, url, null, 5, SecurityEnabled ? 1 : 2, settings, userAgent: "Hello/V");
+            await TestAppSecRequestWithVerifyAsync(_iisFixture.Agent, url, null, 5, 1, settings, userAgent: "Hello/V");
```

Comment:
Snapshots only have had 5 spans with/without security enabled and has been that way for a long time

Reply:
I think that makes sense, I don't believe enabling security generally _adds_ spans these days 🤔